### ISSUE TITLE
usbmount does not rescan xwax library

### ISSUE THREAD

**Issue by obbardc** [pideck/pideck#75]
usbmount is running in a very restricted shell that cannot seem to access the xwax_rescan script.

here is how I currently tried to get it working:

```
echo "env DISPLAY=:0 XAUTHORITY=/home/pi/.Xauthority /usr/share/pideck/xwax_rescan" > /usr/share/pideck/xwax_rescan_wrapper
chmod +x /usr/share/pideck/xwax_rescan_wrapper
ln -s /usr/share/pideck/xwax_rescan_wrapper /etc/usbmount/mount.d/01_xwax_rescan
ln -s /usr/share/pideck/xwax_rescan_wrapper /etc/usbmount/umount.d/01_xwax_rescan
```

**Comment by danielhjames** [comment]
Yep, this does seem to be a regression relative to the pideck 0.1 build. It seems you can also run a script with xauthority from a udev rule: https://askubuntu.com/questions/284224/autorun-a-script-after-i-plugged-or-unplugged-a-usb-device#347971

**Comment by obbardc** [comment]
That is basically what is being done, but new systemd rules lock down the rules entries run from udev.

I don't think there is any easy way to press a key from the udev rule.

I think no longer we need to have better control over xwax anyway, does the osc patchset allow us to remotely scan media?

The osc patch would be great in pideck, anyway!
